### PR TITLE
antag mimes can break their vow early

### PR DESCRIPTION
**Pull request by ShadowDeath6**
uh yeah
fixes https://github.com/yogstation13/yogstation/issues/1966

:cl: ShadowDeath6
tweak: Ratvarian mimes as well as Changeling mimes can now break their vow at any time.
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
Just allow all antag mimes to break it early

**Comment by ShadowDeath6** [comment]
what antags need to talk other than lings and servants though

**Comment by Time-Green** [comment]
idunno, good point.

**Comment by Super3222** [comment]
Traitor Clown gets to not be retarded.
Traitor Mime should be able to break their vow early.

Also it sorta fits in the whole "syndicate agent dressed up as X job to infiltrate the station" or whatever your lore tells you.

File: code/modules/spells/spell_types/mime.dm [hunk 1]
```diff
@@ -46,7 +46,7 @@
 		return
 	if(!ishuman(usr))
 		return
-	if(world.time < 15000) // 25 minutes
+	if(world.time < 15000 && !(usr.mind.changeling) && !(is_servant_of_ratvar(usr)) && !(usr in ticker.mode.traitors) && !(iscultist(usr)) && !(usr in ticker.mode.head.revolutionaries) && !(is_shadow(usr)) && !(is_gangster(usr))) // 25 minutes
```
Comment (Cruix) [inline code/modules/spells/spell_types/mime.dm]: Just do !usr.mind.special_role

Comment (ShadowDeath6) [reply code/modules/spells/spell_types/mime.dm]: oh thanks

**Pull request merged**